sklearn/ensemble/bagging.py

```diff
@@ -95,6 +95,11 @@ def _parallel_build_estimators(n_estimators, ensemble, X, y, sample_weight,
                                                       n_samples, max_features,
                                                       max_samples)
 
+        # Convert sample indices to a masked array; it is the same behavior
```

Comment:
Wow you seem to like semi-colons! Is there a scikit-learn litterature prize I am not aware of ;-) (or maybe this is just me having some bias coming from the French semicolon) ?